docs/addons/mongodb/setup/install.md

```diff
@@ -0,0 +1,110 @@
+
+## Install only YAMLs
+
+Run the following script to install `stash-mongodb` addon as Kubernetes YAMLs.
+
+```console
+curl -fsSL https://github.com/stashed/catalog/raw/master/deploy/script.sh | bash -s -- --catalog=stash-mongodb
+```
+
+</div>
+<div class="tab-pane fade" id="helm" role="tabpanel" aria-labelledby="helm-tab">
+
+## Install as chart release
+
+Run the following script to install `stash-mongodb` addon as a Helm chart.
+
+```console
+curl -fsSL https://github.com/stashed/catalog/raw/master/deploy/chart.sh | bash -s -- --catalog=stash-mongodb
+```
+
+</div>
+</div>
+
+## Verify Installation
+
+After installation is completed, this addon will create `mongodb-backup-*` and `mongodb-restore-*` Functions and Tasks for all supported MongoDB versions. To verify, run the following command:
+
+```console
+$ kubectl get functions.stash.appscode.com
+NAME                    AGE
+mongodb-backup-4.1      20s
+mongodb-backup-4.0      20s
+mongodb-backup-3.6      19s
+mongodb-backup-3.4      20s
+mongodb-restore-4.1     20s
+mongodb-restore-4.0     20s
+mongodb-restore-3.6     19s
+mongodb-restore-3.4     20s
+pvc-backup              7h6m
+pvc-restore             7h6m
+update-status           7h6m
+```
+
+Also, verify that the `Task` have been created.
+
+```console
+$ kubectl get tasks.stash.appscode.com
+NAME                    AGE
+mongodb-backup-4.1      2m7s
+mongodb-backup-4.0      2m7s
+mongodb-backup-3.6      2m6s
+mongodb-backup-3.4      2m7s
+mongodb-restore-4.1     2m7s
+mongodb-restore-4.0     2m7s
+mongodb-restore-3.6     2m6s
+mongodb-restore-3.4     2m7s
+pvc-backup              7h7m
+pvc-restore             7h7m
+```
+
+Now, Stash is ready to backup MongoDB database.
+
+## Customizing Installation
+
+In order to install `Function` and `Task` only for a specific MongoDB version, use `--version` flag to specify the desired database version.
+
+```console
+curl -fsSL https://github.com/stashed/catalog/raw/master/deploy/chart.sh | bash -s -- --catalog=stash-mongodb --version=3.6
+```
+
+The flowing flags are available for customizing MongoDB addon installation:
+
+| Flag                | Usage                                                                                          |
+| ------------------- | ---------------------------------------------------------------------------------------------- |
+| `--version`         | Specify a specific version of a specific addon to install. Use it along with `--catalog` flag. |
+| `--docker-registry` | Specify the docker registry to use to pull respective addon images. Default Value: `stashed`.  |
+| `--image`           | Specify the name of the docker image to use for respective addons.                             |
+| `--image-tag`       | Specify the tag of the docker image to use for respective addon.                               |
+| `--mg-backup-args`  | Specify optional arguments to pass to `mongodump` command during backup.                       |
```

Comment:
Why do we pass these args at the installer level? Shouldn't these be used at BackupConfiguration level?